libs/sdk-backend-mockingbird/src/recordedBackend/index.ts

```diff
@@ -64,8 +64,14 @@ import {
     RecordedWorkspaceUsersQuery,
 } from "./userManagement";
 
-const defaultConfig: RecordedBackendConfig = {
+/**
+ * @internal
+ */
+export const defaultConfig: RecordedBackendConfig = {
     hostname: "test",
+    globalSettings: {
+        enableKPIDashboardDrillFromAttribute: true,
```

Comment:
This should be specified in the individual tests, no? Why is it set for the whole backend?

Reply:
In other words any changes to mockingbird should not be necessary.